src/_guides/language/language-tour.md

```diff
@@ -3698,6 +3701,37 @@ try {
 }
 {% endprettify %}
 
+You can use `await` multiple times in an async function.
+For example, the following code waits three times
+for the results of functions:
+
+<?code-excerpt "misc/lib/language_tour/async.dart (repeated-await)"?>
+{% prettify dart %}
+var entrypoint = await findEntrypoint();
+var exitCode = await runExecutable(entrypoint, args);
+await flushThenExit(exitCode);
+{% endprettify %}
+
+In <code>await <em>expression</em></code>,
+the value of <code><em>expression</em></code> is usually a Future;
+if it isn't, then the value is automatically wrapped in a Future.
+This Future object indicates a promise to return an object.
+The value of <code>await <em>expression</em></code> is that returned object.
+The await expression makes execution pause until that object is available.
+
+**If you get a compile-time error when using `await`,
+make sure `await` is in an async function.**
+For example, to use `await` in your app's `main()` function,
+the body of `main()` must be marked as `async`:
+
+<?code-excerpt "misc/lib/language_tour/async.dart (main)"?>
+{% prettify dart %}
+Future main() async {
+  checkVersion();
+  print('In main: version is ${await lookUpVersion()}');
+}
+{% endprettify %}
```

Comment:
At a quick glance it is hard to find the `await` inside the string interpolation expression. I suggest highlighting `async` and `await` in this excerpt (similar to what is done in previous excerpts).